src/Microsoft.Health.Fhir.Api/Features/Bundle/FhirTransactionFailedException.cs

```diff
@@ -42,5 +42,12 @@ public FhirTransactionFailedException(string message, HttpStatusCode httpStatusC
         }
 
         public HttpStatusCode ResponseStatusCode { get; }
+
+        public bool IsErrorCausedDueClientFailure()
```

Comment:
Does this include 408 timeouts when, for example when SQL timesout?

Reply:
Nope. It'll not. It'll only be true in case an exception is raised, and the operation outcome is filled for a record. Do you see need for that?